quickstarts/istio/config.yml

```diff
@@ -0,0 +1,56 @@
+# Name of the quickstart (required)
+name: istio-quickstart
```

Comment:
We've been recommending to not use the term "quickstart" in a quickstart as it's redundant.